src/features/pages/home/live-pricing/market-tabs/tab-style.module.scss

```
@@ -0,0 +1,39 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+.tab_container {
+    overflow: scroll;
+    padding: 2rem 0px;
+    align-items: center;
+    &::-webkit-scrollbar {
+        display: none;
+    }
+
+    @include breakpoints(tablet) { 
+    overflow-x: auto;
+    justify-content: center;
+    padding: 2rem;
+}
+}
+.inactive_button{
+    white-space: nowrap;
+    border-inline: none;
+    border-block-end: 2px solid $color-grey-8;
+    border-radius: unset;
```

can be passed via props as
```suggestion
    radius="0x"
```